src/Migrator.php

```diff
@@ -18,7 +18,8 @@ public function __construct(
         private ConnectionInterface $db,
         private MigrationInformerInterface $informer,
         private string $historyTable = '{{%migration}}',
-        private ?int $migrationNameLimit = 180
+        private ?int $migrationNameLimit = 180,
+        private int $maxSqlOutputLength = 0,
```

Comment:
```suggestion
        private ?int $maxSqlOutputLength = null,
```

This consistent with `$migrationNameLimit`.

Reply:
For `$migrationNameLimit` `null` has meaming but for `$maxSqlOutputLength` I think no needed.

Reply:
Hm... Does it make sense not to output SQL?

Reply:
The same as put 1 there. User can decide for what it is there.

Reply:
> Do not output sql if $maxSqlOutputLength equals 0 and do not limit if null

It's OK for me.

Reply:
Also fixed #214